config/application.rb

```diff
@@ -7,20 +7,33 @@
 Bundler.require(*Rails.groups)
 
 module Flashcards
-  class Application < Rails::Application
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
-
-    # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
-    # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
-    # config.time_zone = 'Central Time (US & Canada)'
-
-    # The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
-    # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
-    # config.i18n.default_locale = :de
-
-    # Do not swallow errors in after_commit/after_rollback callbacks.
-    config.active_record.raise_in_transactional_callbacks = true
-  end
+    class Application < Rails::Application
+# Settings in config/environments/* take precedence over those specified here.
+# Application configuration should go into files in config/initializers
+# -- all .rb files in that directory are automatically loaded.
+
+# Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
+# Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
+# config.time_zone = 'Central Time (US & Canada)'
+
+# The default locale is :en and all translations from config/locales/*.rb,yml are auto loaded.
+# config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
+# config.i18n.default_locale = :de
+
+# Do not swallow errors in after_commit/after_rollback callbacks.
+
+ActionView::Base.field_error_proc = Proc.new do |html_tag, instance|
```

Comment:
Unused block argument - `instance`. If it's necessary, use `_` or `_instance` as an argument name to indicate that it won't be used.<br>Use 2 (not -4) spaces for indentation.